virttest/qemu_vm.py

```diff
@@ -1694,6 +1709,13 @@ def sort_key(dev):
                                     numa_mem, numa_cpus, numa_nodeid)
             devices.insert(StrDev('numa', cmdline=cmdline))
 
+        for numa_node in params.objects("guest_numa_nodes"):
+            numa_params = params.object_params(numa_node)
+            numa_nodeid = numa_params.get("numa_nodeid")
+            numa_dist = ast.literal_eval(numa_params.get("numa_dist", "[]"))
```

Comment:
Would you please add a comment in cfg to explain the new param 'numa_dist'?

Reply:
Add comment in base.cfg, thanks!